benchmarking/saving_fingerprints_to_npy.py

```diff
@@ -0,0 +1,165 @@
+import logging
+import os
+from time import time
+
+import numpy as np
+import pandas as pd
+from ogb.graphproppred import GraphPropPredDataset
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import roc_auc_score
+from sklearn.preprocessing import MinMaxScaler
+
+from skfp.fingerprints import *
+from skfp.preprocessing import ConformerGenerator, MolFromSmilesTransformer
+
+dataset_names = [
+    "ogbg-molhiv",
+    "ogbg-molbace",
+    "ogbg-molbbbp",
+]
+
+property_names = [
+    "HIV_active",
+    "Class",
+    "p_np",
+]
+
+fp_names = [
```

Comment:
A few naming conventions and fixing typos:
- Autocorr -> Autocorrelation (note that we have 2 variants, 2D and 3D there)
- ERG -> ErG
- Getway -> GETAWAY
- MORDRED -> Mordred
- MORSE -> MoRSE
- Phsiochemical-Properties -> Physiochemical Properties
- Topological-Torsion -> Topological Torsion
- Whim -> WHIM

Reply:
I would suggest using class.__name__ instead. It will simplify reading saved files and eliminate errors such as typos.

Reply:
Agreed